geopandas/geoseries.py

```diff
@@ -629,6 +634,11 @@ def _constructor_from_mgr(self, mgr, axes):
     def _constructor_expanddim(self):
         return _geoseries_expanddim
 
+    def _constructor_expanddim_from_mgr(self, mgr, axes):
+        df = pd.DataFrame._from_mgr(mgr, axes)
+        existing_name = mgr.axes[0]
```

Comment:
I would have expected this needs to be `mgr.axes[0][0]` (first [0] to get the first axis (i.e. the columns' Index, in the BlockManager twisted order of (columns, index)) and the second [0] to get the single scalar name out of that Index object). 
But if the tests are not failing ..?

Reply:
I'll check, I think this is the same point as the missing coverage above, that this argument/ value might be redundant.